packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -139,21 +140,16 @@ const InlineMediaIndicator = styled.div`
 `;
 
 export const Headline = styled.h3`
-  ${props => (props.script ? getPica(props.script) : '')};
+  ${({ script, topStory }) =>
+    script && topStory ? getParagon(script) : getPica(script)}}
```

Comment:
```suggestion
    script && (topStory ? getParagon(script) : getPica(script))}
```
I think this is also needed, but definitely looks better :+1: